unfoldingWord/translationCore
# Refactor Project Loading

#### This pull request addresses:
This PR simplifies the loading process for a project in tC, mostly dealing with getDataActions.js
#1825 

#### How to test this pull request:
To test this pr try opening different projects and make sure you can get to the modal showing tN and tW. Tools are not currently piped to load everything atm.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1862)
